cmd/arena/commands/top_node.go

```diff
@@ -62,11 +64,47 @@ func NewTopNodeCommand() *cobra.Command {
 				fmt.Println(err)
 				os.Exit(1)
 			}
+			tmp := false
```

Comment:
we need the meaningful variable definition here.

Reply:
Thanks for your guidance, I have modified these places, please check it again.